index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Tribute_page</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <div id="main">
+        <div id="title">
+            <h1>Dr. Norman Borlaug</h1>
+            <h3>The man who saved a billion lives</h3>
+        </div>
+        <div class="img-div">
+            <img id="img-caption" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt="photography">
+            <h3>Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part
+                of his life-long war on hunger.</h3>
+        </div>
+        <div class="p">
+            <h2>Here's a time line of Dr. Borlaug's life:</h2>
+        </div>
+        <div class="list">
```

Comment:
Please use semantic tags Instead of <div> tag . You should use <main> tag instead of <div id=main>

Reply:
Good job 👍 in identifying the proper use of semantic tags.